app/models/rebate_form.rb

```diff
@@ -2,8 +2,8 @@
 
 class RebateForm < ApplicationRecord
   has_many :signatures, dependent: :destroy
-  belongs_to :property, required: true
-  belongs_to :batch, required: false
+  belongs_to :property, optional: false
```

Comment:
isn't this the default on rails 5 now?

Reply:
Yeah, I think it might be. But since we were explicitly stating when it's true I figured it wasn't a big deal to explicitly state it when it's false.